docs/serving/samples/multi-container/README.md

```diff
@@ -0,0 +1,250 @@
+A simple web app written in Go that you can use for multi container testing.
+
+Follow the steps below to create the sample code and then deploy the app to your
+cluster. You can also download a working copy of the sample, by running the
+following command:
+
+```shell
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+```
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+- Make sure multi-container flag is enabled as part of `config-features` configmap.
+
+## Recreating the sample code
+
+Multi container has more than one container. For testing, we will create two containers: a serving container and a sidecar container.
+The `multi-container` directory already exist with predefined code and dockerfile for serving and sidecar container with service yaml.
+If user wants to update code, dockerfile or service yaml they can follow below steps.
+
+### Serving Container
+1. `cd knative-docs/docs/serving/samples/multi-container/servingcontainer`
```

Comment:
Something like:

1. After you have cloned the sample repository, navigate to the `servingcontainer` directory:
   ```
   cd knative-docs/docs/serving/samples/multi-container/servingcontainer
   ```